src/material/bottom-sheet/bottom-sheet.md

```diff
@@ -50,6 +50,7 @@ export class HobbitSheet {
 ```
 
 ### Configuring bottom sheet content via `entryComponents`
+**Since 9.0.0. With Ivy, this is no longer necessary.**
```

Comment:
Sorry for nitpicking, but what do you think about rewording it to something like "This is only necessary if your project is using ViewEngine. It doesn't have to be specified for projects running on Ivy."

Reply:
agree. it's better
I'll update the sentence